jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -245,6 +247,40 @@ public JavaContainerBuilder addDependencies(Path... dependencyFiles) throws IOEx
     return addDependencies(Arrays.asList(dependencyFiles));
   }
 
+  /**
+   * Adds the contents of a dependency JAR directory to the image. Duplicate JAR filenames are
```

Comment:
This seems confusing: we now have 4 variants of `addDependencies`, but some handle directories whereas others don't.

Reply:
Hmm, yeah I think I'll remove the two new methods. This can be done just taking a list of files.